ingest/defaults/config.yaml

```diff
@@ -43,7 +42,6 @@ curate:
     isolate-lineage: strain
     geo-region: region
     geo-location: location
-    isolate-collection-date: date
```

Comment:
I see, you're parsing `isolate-collection-date` from GenBank directly using `ingest/scripts/extract-genbank-genotypes.py` where it automatically picks the best date from `/collection-date` or if there's an epi-week or just year from `/strain` or `/isolate=(.*)`. 

https://github.com/nextstrain/rubella/blob/616ac150b04f5e63f1afc9509117cefacd604fc4/ingest/scripts/extract-genbank-genotypes.py#L117-L132

Non-blocking suggestion to call the script `extract-genbank-annotations.py` since you're parsing out "genotype" and "date"?

Reply:
Renamed.